src/components/AstronautsPage/actions/actionCreators.ts

```diff
@@ -0,0 +1,16 @@
+import { AstronautsActions } from './actionTypes';
+import type { Astronauts } from '../types/Astronauts';
+
+export const astronautsRequestAction = () => ({
+  type: AstronautsActions.FETCH_ASTRONAUTS_REQUEST,
+});
+
+export const astronautsSuccessAction = (astronauts: Astronauts[]) => ({
+  type: AstronautsActions.FETCH_ASTRONAUTS_SUCCESS,
+  payload: astronauts,
+});
+
+export const astronautseErorAction = (error: string) => ({
+  type: AstronautsActions.FETCH_ASTRONAUTS_ERROR,
+  payload: error,
+});
```

Comment:
1. Provide types to action creators
2. typo eror, astronautse
3. Rename action creators as action names
4. Unsuccess action usually called as Failure

Reply:
1. Done
2. Fixed
3. Renamed
4. Renamed to fectchAstronautsFailure